backend/src/main/java/capstone/herbs/user/UserDocument.java

```diff
@@ -0,0 +1,16 @@
+package capstone.herbs.user;
+
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection="users")
+@Data
+public class UserDocument {
+    @Id
+    String Id;
```

Comment:
```suggestion
    String id;
```

variables in lowercase